app/models/proposal.rb

```diff
@@ -15,6 +15,7 @@ class Proposal < ActiveRecord::Base
   include ActsAsParanoidAliases
 
   RETIRE_OPTIONS = %w(duplicated started unfeasible done other)
+  VALID_DOMAINS = %w(www.youtube.com youtube.com youtu.be www.vimeo.com vimeo.com)
```

Comment:
`VALID_VIDEO_DOMAINS` would be more descriptive as a constant name, what do you think?